ansible/pmm2/post-build-actions.yml

```diff
@@ -25,11 +25,20 @@
 
     # TODO https://jira.percona.com/browse/PMM-4991
     - name: Detect distribution        | Create '/srv/pmm-distribution' file for AMI
-      when: ansible_virtualization_type == "xen" or ansible_virtualization_type == "kvm"
+      when: >
+        ( ansible_virtualization_type == "xen"
+        or ansible_virtualization_type == "kvm" )
+        and ansible_system_vendor != "DigitalOcean"
       copy:
         content: "ami"
         dest: /srv/pmm-distribution
 
+    - name: Detect distribution        | Create '/srv/pmm-distribution' file for DigitalOcean
+      when: ansible_system_vendor == "DigitalOcean"
+      copy:
+        content: "digitalocean"
+        dest: /srv/pmm-distribution
+
```

Comment:
We'll need to add this option to telemetry before merging this PR:
https://github.com/percona/pmm-managed/blob/9be983ec7cb5c2b90979bed7c0105a1a0fd222ac/services/telemetry/telemetry.go#L126

Reply:
I checked and we don't have custom pmm-distribution for Azure image... I'll discuss with the team that will be better to use here.

Reply:
There are two things that make the DigitalOcean environment different from the other cloud providers, and require special treatment in the ansible role:
- Unlike AWS (and Azure I presume), DO doesn't have the capability to automatically mount a block storage volume on a Marketplace droplet.
- We want to include the helper script to set up access to a user's DBaaS instances.